pkg/csi/service/common/commonco/k8sorchestrator/k8sorchestrator.go

```diff
@@ -105,34 +105,34 @@ func (m *volumeIDToPvcMap) get(volumeHandle string) string {
 	return m.items[volumeHandle]
 }
 
-// Map of volumeID to the list of node names in the cluster.
-// Key is the volume ID and value is the list of published nodes for the volume
+// Map of volumeName to the list of node names in the cluster.
+// Key is the volume name and value is the list of published nodes for the volume
 // The methods to add, remove and get entries from the map in a threadsafe
 // manner are defined.
-type volumeIDToNodesMap struct {
+type volumeNameToNodesMap struct {
 	*sync.RWMutex
 	items map[string][]string
 }
 
-// Adds an entry to volumeIDToNodesMap in a thread safe manner.
-func (m *volumeIDToNodesMap) add(volumeID string, nodes []string) {
+// Adds an entry to volumeNameToNodesMap in a thread safe manner.
+func (m *volumeNameToNodesMap) add(volumeName string, nodes []string) {
 	m.Lock()
 	defer m.Unlock()
-	m.items[volumeID] = nodes
+	m.items[volumeName] = nodes
 }
 
-// Removes a volumeID from the volumeIDToNodesMap in a thread safe manner.
-func (m *volumeIDToNodesMap) remove(volumeID string) {
+// Removes a volumeName from the volumeNameToNodesMap in a thread safe manner.
+func (m *volumeNameToNodesMap) remove(volumeName string) {
 	m.Lock()
 	defer m.Unlock()
-	delete(m.items, volumeID)
+	delete(m.items, volumeName)
 }
 
-// Returns the list of published nodes for the given volumeID in a thread safe manner.
-func (m *volumeIDToNodesMap) get(volumeID string) []string {
+// Returns the list of published nodes for the given volumeName in a thread safe manner.
```

Comment:
Can you use pvName instead of volumeName for better readability

Reply:
done